client/my-sites/purchases/subscriptions/account-level-advertising-links.tsx

```diff
@@ -1,21 +1,32 @@
+import { isEnabled } from '@automattic/calypso-config';
 import { CompactCard } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
 import { PromoteWidgetStatus, usePromoteWidget } from 'calypso/lib/promote-post';
 import { useSelector } from 'calypso/state';
-import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getSiteOption } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
 export default function AccountLevelAdvertisingLinks() {
 	const translate = useTranslate();
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
 	const shouldShowAdvertisingOption = usePromoteWidget() === PromoteWidgetStatus.ENABLED;
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const adminInterface = useSelector( ( state ) =>
+		getSiteOption( state, siteId, 'wpcom_admin_interface' )
+	);
 
 	if ( ! shouldShowAdvertisingOption || ! selectedSiteSlug ) {
 		return null;
 	}
 
+	const advertisingUrl =
```

Comment:
nit: since we're doing the check in three places now, it might be justified to create a helper like `useAdvertisingUrl()` that would also handle loading `adminInterface` and `siteId`

Reply:
Sounds good! I created client/my-sites/advertising/useAdvertisingUrl.jsx, but what do you think about the location?